packages/shared/components/popups/StakingManager.svelte

```diff
@@ -240,6 +257,14 @@
                         <div class="bg-green-100 rounded-2xl">
                             <Icon icon="success-check" width="19" height="19" classes="text-white" />
                         </div>
+                    {:else if getAccountParticipationAbility(account) === AccountParticipationAbility.NoWillNotReachMinAirdrop}
```

Comment:
This statement will always resolve false because the condition is not included in the parent if statement.